### DISCUSSION TITLE
Add Template repositories.

### DISCUSSION THREAD

**Discussion by dibu28** [louislam/dockge#289] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Add feature so that the user can point url to template repository list and can see the list of container templates from this repository list and can quickly start spining up containers from a collection of a precreated templates.

### ✔️ Solution

Also a default repository needed. Or we can use template repositiries from other projects like Portainer etc. And community repositories can also appear over time.

**Comment by Mugane** [comment]
This is a blocking feature for me, I can't even tolerate Portainer's poorly customizable template system. I need to be able to build interfaces to set up compose templates, with actual fieldsets, drop-down/selects, and boolean logic (parameters that depend on the value of other parameters).

**Comment by wsw70** [comment]
I would be happy with even a basic template, copy-paste-style to gather all the fields I need for each of my containers. Right now I copy a basic container `compose.yml` but this is really artisanal.
